src/FormulateInput.vue

```diff
@@ -127,7 +127,7 @@ export default {
     formulateValue: {
       default: ''
     },
-    value: {
+    modelValue: {
```

Comment:
Should this also change the API from an end user perspective? Ie should they be using `<FormulateInput model-value="whatever"`? This causes some issues with the annoying `$.vnode.props` stuff we have to do all over at the moment.

Reply:
I personally would prefer to conform to the html api and not the vue api. It offers an easier entry into the library api as most people know the default api already.

Reply:
@axwalker — the `value` is not the same as the `modelValue` though. Right now the `modelValue` is mapped to `formulateModel` while `value` is a non-bound value hydrator (to more closely model native HTML). I definitely think we should keep that the same. I guess one question is can we re-map `v-model` values in Vue 3 [like you can in Vue 2](https://vuejs.org/v2/api/#model)

@fseemann — there is no native api for `v-model` — but if you're referring to the `value` attribute/prop then i agree. And in the current version that already follows standard HTML api in Vue Formulate (default hydration, highest priority).